woocommerce/woocommerce-paypal-payments
# Apple Pay: Only render the button markup when Apple Pay is available (3861)

### Description

This PR adds availability checking for the Apple Pay button to only render the button markup when Apple Pay is available.

### Steps to test

1. Enable Apply Pay and test using Safari.
2. Open the block checkout (on the page) and verify that the Apple Pay button displays correctly.
3. Open the block checkout with a different browser. Ensure the button doesn't render and the rest of the buttons display in a correct layout.
 
### Screenshots

|Before|After|
|-|-|
|<img width="792" alt="Block_Checkout_–_WooCommerce_PayPal_Payments" src="https://github.com/user-attachments/assets/b0b8efee-184a-4a62-819f-23db1f134b35">|<img width="790" alt="Block_Checkout_–_WooCommerce_PayPal_Payments" src="https://github.com/user-attachments/assets/da244c35-ac91-450a-a2be-a7e9eaded139">|
